test/e2e/example-test.json

```diff
@@ -0,0 +1,10 @@
+{
+	"name": "test-example",
+	"namespace": "testkube",
+	"description": "Example simple test orchestration",
+	"steps": [
+		{"type": "executeScript", "namespace": "testkube", "name": "test1"},
```

Comment:
do we only have executeScript and delay "actions" for now?

Reply:
yes